File: 8 kyu/Calculate BMI/solution.js
function bmi(weight, height) {
  const calc = weight / height ** 2;
  switch (true) {
    case calc <= 18.5:
      return 'Underweight';
    case calc <= 25.0:
      return 'Normal';
    case calc <= 30.0:
      return 'Overweight';
    case calc > 30:
      return 'Obese';
  }
}

const result = bmi(80, 1.8);
console.log(result); // 🐣 "Normal"

File: 8 kyu/Calculate BMI/solution.cs
using System;

public class Kata
{
  public static string Bmi(double weight, double height)
  {
    var calc = (weight / Math.Pow(height,2));
    
    if(calc <= 18.5) return "Underweight";
    if(calc <= 25.0) return "Normal";
    if(calc <= 30.0) return "Overweight";
    return "Obese";
  }
}
